FWCore/Services/plugins/CheckTransitions.cc

```diff
@@ -276,7 +310,17 @@ void CheckTransitions::postEndJob() {
     ++itOS;
   }
 
-  if (orderedSeen.size() != orderedExpected.size()) {
+  if (not(seenGlobalBeginLumi == seenStreamBeginLumi)) {
```

Comment:
How about
```suggestion
  if (seenGlobalBeginLumi != seenStreamBeginLumi) {
```
? Or did you have reason avoid that?

(same for the if statement few lines below)

Reply:
Should have done it that way in the first place. Thanks. Done.